src/nova-components/CreateFavorites/resources/js/components/Tool.vue

```diff
@@ -3,49 +3,117 @@
     <div class="input-container">
       <h1>Create Favorite</h1>
       <div class="input-container-background">
-        <text-field
-          :isRequired="true"
-          :type="textInput"
-          :label="title"
-          :placeholder="title"
-          :error="errorTitle"
-          :sameError="errorSameTitle"
-          :changeInput="changeInput"
-          :input="inputTitle"
-        />
-        <text-field
-          :isRequired="false"
-          :type="textInput"
-          :label="author"
-          :placeholder="author"
-          :changeInput="changeInput"
-          :input="inputAuthor"
-        />
-        <text-field
-          :isRequired="true"
-          :type="urlInput"
-          :label="source"
-          :placeholder="source"
-          :error="errorSource"
-          :validationError="validationErrorSource"
-          :changeInput="changeInput"
-          :input="inputSource"
-        />
-        <text-field
-          :isRequired="true"
-          :type="dateInput"
-          :label="postDate"
-          :placeholder="postDate"
-          :error="errorPostDate"
-          :changeInput="changeInput"
-          :input="inputPostDate"
-        />
-        <dropdown-field
-          :label="user"
-          :error="errorUser"
-          :changeInput="changeInput"
-          :input="inputUser"
-        />
+        <div class="field-container">
+          <div class="label-spacing">
+            <label-text :label="title" :isRequired="true" />
+          </div>
+          <div class="input-spacing">
+            <text-input
+              :type="textInput"
+              :error="errorTitle"
+              :sameError="errorSameTitle"
+              :placeholder="title"
+              :changeInput="changeInput"
+              :input="inputTitle"
+            />
+            <error-label
+              v-if="errorTitle"
+              :label="title"
+              :errorType="emptyError"
+            />
+            <error-label
+              v-else-if="errorSameTitle"
+              :label="title"
+              :errorType="sameError"
+            />
```

Comment:
don't use multiple error-label nodes, find a way to do it with computed attr and one node